### PR TITLE
Fix name of cross_pollination_stats table

### PR DESCRIPTION
**Pull request by mpherman2**

### CODE REVIEW

File: configs/test/bigquery/datasets.yaml [hunk 1]
```diff
@@ -230,7 +230,7 @@ resources:
     properties:
       datasetId: $(ref.main-dataset.datasetReference.datasetId)
       tableReference:
-        tableId: cross-pollination-statistics
+        tableId: cross_pollination_statistics
       timePartitioning:
         type: DAY
       schema:
```
Comment (mbarbella-chromium) [inline configs/test/bigquery/datasets.yaml]: Could you also indent the children under fields: below here for consistency with the other tables.

Comment (mpherman2) [reply configs/test/bigquery/datasets.yaml]: Didn't notice, thanks for pointing that out. Done

**Pull request merged**